students/Hebdowska_Grazyna/lesson_06/the_number_of_distinct_words_in_some_test.py

```diff
@@ -0,0 +1,19 @@
+def distings_words(some_tesxt):
+    new_word = {}
+    for i in range(len(some_tesxt)):
+        new_text = some_tesxt[i].split(' ')
+        if '' in new_text:
+            new_text.remove('')
+        for n in new_text:
+            if n not in new_word.keys():
+                new_word[n] = 1
+    number_of_words = len(new_word)
+    return number_of_words
+
+
+nn = int(input())
```

Comment:
Please try to not use abbreviation for variable names
e.g.
nn ->  number_of_lines